apps/sms/test/marionette/lib/views/new-message/accessors.js

```diff
@@ -14,6 +14,9 @@ var SELECTORS = Object.freeze({
   subjectInput: '.subject-composer-input',
   sendButton: '#messages-send-button',
   attachButton: '#messages-attach-button',
+  headerActionButton: '#messages-options-button',
```

Comment:
Yeah, because this button is in ```ShadowDOM```[1], that's why we can access it only after calling ```switchToShadowRoot``` ;)

That's is why I recommended you to run the app and see what's going on in real using Dev Tools inspector, and it will be useful for you as well, so that you know how Messages app looks like :)

[1] http://mdn.io/shadowdom